packages/core/src/lib/resource-pill.svelte

```diff
@@ -0,0 +1,69 @@
+<!--
+@component
+  
+For displaying a list of items.
+
+```svelte
+<ResourcePill value="BeagleBone" on:remove={onRemove} />
+```
+-->
+<svelte:options immutable />
+
+<script lang="ts">
+import { Icon, type IconName } from '$lib';
+import cx from 'classnames';
+
+import { createEventDispatcher } from 'svelte';
+
+/** The text in the resource pill. */
+export let value = '';
```

Comment:
I would _really_ prefer this to be a default `<slot />` rather than a `value` prop

Reply:
Being that I've moved everything into `Pill` I think we'll have to do that as a follow up as it would require updates in APP and RDK.

Reply:
Follow up ticket: https://viam.atlassian.net/browse/APP-3275